README.md

```diff
@@ -70,7 +70,7 @@ jobs:
           fail_silently: false
 
       - if: |-
-          steps.analyze-code-security-scc.outputs.iac_scan_result_sarif_path != ''
+          (steps.analyze-code-security-scc.outputs.iac_scan_result_sarif_path != '') && (success() || failure())
```

Comment:
You can use the `always()` function instead to clean this up: https://docs.github.com/en/actions/learn-github-actions/expressions#always

Reply:
Done

Reply:
@kishorekgarg @verbanicm 

There Is a fundamental problem with this approach that even for cancelled runs it will run which is not desired. We only wanted to keep it for scenarios where above steps are either successful or failed.

Reply:
That is good context, good call out. I would instead use `if: ${{ !cancelled() && steps.analyze-code-security-scc.outputs.iac_scan_result_sarif_path != '' }}` as recommended by GitHub. See the WARNING text here: https://docs.github.com/en/actions/learn-github-actions/expressions#always

I would also document this in a comment inline so user know why it was created this way for future reference.